cli/src/commands/init.rs

```diff
@@ -78,14 +78,24 @@ pub struct Opts {
 
 impl Opts {
 	pub async fn execute(&self, term: &Term) -> GlobalResult<()> {
-		// Remove legacy `.rivet` dir if exists
+		// Remove legacy `.rivet/config.[yaml|toml|json]` files if they exists
 		let legacy_project_meta_path = paths::project_root()?.join(".rivet");
-		if fs::metadata(&legacy_project_meta_path).await.is_ok() {
+		if let Ok(mut entries) = fs::read_dir(&legacy_project_meta_path).await {
 			term::status::warn(
 				"Deleting legacy project metadata",
-				".rivet/ folder is moved to a global config",
+				".rivet/config.[yaml|toml|json] file is moved to a global config",
 			);
-			fs::remove_dir_all(&legacy_project_meta_path).await?;
+			while let Some(entry) = entries.next_entry().await? {
+				if let Some(file_name) = entry.file_name().to_str() {
+					if file_name.starts_with("config")
+						&& [".yaml", ".yml", ".toml", "json"]
```

Comment:
The CLI was only ever configured to use `config.toml` fyi, that path was hardcoded and didn't use the `config` crate.